agent/agent.go

```diff
@@ -180,7 +190,8 @@ func (a *Agent) k8sPodUpHandle(netif NetIf) error {
 		return agentError(err)
 	}
 	log.Print("Agent: provisioning firewall")
-	if err := provisionFirewallRules(netif, a); err != nil {
+
+	if err := provisionK8SFirewallRules(netReq, a, fullIsolation); err != nil {
```

Comment:
I think `fullIsolation` is too explicit. Reader might assume there is a partial isolation and other flavours of isolation.
I'd call it just `isolation` or `enableNamespaceIsolation`.